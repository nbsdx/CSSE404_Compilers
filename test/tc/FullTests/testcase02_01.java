// checking case sensitivity
class Bar {
	public static void main(String[] args) {
		int x = 2;
		int y = 3;
		boolean X = false;
		Bar something = new Bar();
		y = X + x;
	}
}