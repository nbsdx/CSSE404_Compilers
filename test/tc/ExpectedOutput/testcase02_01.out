Left argument of type boolean does not match expected type int for operator PLUS
